src/stories/common/ButtonWithIcon.stories.js

```diff
@@ -0,0 +1,57 @@
+import ButtonWithIcon from "../../components/common/ButtonWithIcon";
+import {faStar} from "@fortawesome/free-solid-svg-icons";
+
+export default {
+    title: 'Common/ButtonWithIcon',
+    component: ButtonWithIcon,
+    argTypes: {
+        type: {
+            control: { type: 'select' },
+            options: ['primary', 'secondary', 'success', 'danger']
+        },
+        onClick: { action: 'clicked' },
+    },
+};
+
+const Template = (args) => ({
+    components: { ButtonWithIcon },
+    setup() {
+        return { args };
```

Comment:
When you want to use `faStar` you need to refernce it in this setup. See this [example](https://github.com/Galleme/frontend/blob/main/src/components/ExampleComponent.vue#L31).